docker.sh

```diff
@@ -77,6 +68,9 @@ function runBuild {
 
 function runInBackground {
     export IMAGE_VERSION=$1
+
+    echo "${IMAGE_VERSION}"
```

Comment:
To remove. This variable is printed [here](https://github.com/PGSSoft/php-docker-skeleton/blob/master/docker.sh#L13)